woocommerce/google-listings-and-ads
# Improve post-onboarding: Show promotion for Google Ads campaign on the Product Feed page

### Changes proposed in this Pull Request:



Closes #2539 

_Replace this with a good description of your changes & reasoning._


### Screenshots:

####  When there is Product but no campaign:  ( Show the notice )
![no-campaign](https://github.com/user-attachments/assets/1c29ef18-d145-41c4-b84c-cae1edd39ba7)
![no-campaign-yes-notice](https://github.com/user-attachments/assets/2f0d0675-6bb8-43e4-92e4-2f8df5f382a9)

#### When there is Product and a campaign:  ( Don't show the notice )
![campaign](https://github.com/user-attachments/assets/a07b83c8-24b1-4432-a88a-4060efd7d132)
![when-there-is-campaign](https://github.com/user-attachments/assets/b16b1c3b-7b5a-4066-adf6-16c09dc44b5f)



### Detailed test instructions:


1. Create a Simple Product, sync to merchant account.
2. Visit Dashboard > Product Feed
3. If there is one active ( approved or partially approved) product and no campaign, notice will be shown. Check screenshots.
4. Click on the button and add campaign. Once added revisit Product Feed page and this section should disappear.

### Additional details:
Further more this ticket address a bug in `useAdsCampaign` https://github.com/woocommerce/google-listings-and-ads/issues/2539#issuecomment-2397847396, which required certain E2E to be adjusted. 
 

### Changelog entry

> Add - Show promotion for Google Ads campaign on the Product Feed page.
